pkg/clusteragent/admission/common/const.go

```diff
@@ -8,4 +8,10 @@
 
 package common
 
-const EnabledLabelKey = "admission.datadoghq.com/enabled"
+const (
+	// EnabledLabelKey pod label to disable/enable mutations at the pod level.
+	EnabledLabelKey = "admission.datadoghq.com/enabled"
+
+	// InjectionModeLabelKey pod label to chose the config injection at the pod level.
+	InjectionModeLabelKey = "admission.datadoghq.com/config.mode"
```

Comment:
maybe it can be an annotation

Reply:
It's for consistency, `admission.datadoghq.com/enabled` is a label already 🤔